SVGLayer.m

```diff
@@ -86,13 +92,19 @@ - (void)setPaths:(NSArray<SVGBezierPath*> *)paths
 
 - (void)setFillColor:(CGColorRef)aColor
 {
-    _fillColor = aColor;
+    CGColorRef previousColor = _fillColor;
+    _fillColor = CGColorRetain(aColor);
+    CGColorRelease(previousColor);
```

Comment:
This is nitpicking, but I'm very allergic to unnecessary variables. And writing it like below would eliminate the need for one:
```objc
CGColorRetain(aColor);
CGColorRelease(_fillColor);
_fillColor = aColor;
```

Reply:
Sure.